src/pages/home/report/ReportActionItem.js

```diff
@@ -437,11 +437,11 @@ function ReportActionItem(props) {
     }
 
     const hasErrors = !_.isEmpty(props.action.errors);
-    const whisperedTo = props.action.whisperedTo || [];
-    const isWhisper = whisperedTo.length > 0;
-    const isMultipleParticipant = whisperedTo.length > 1;
-    const isWhisperOnlyVisibleByUser = isWhisper && ReportUtils.isCurrentUserTheOnlyParticipant(whisperedTo);
-    const whisperedToPersonalDetails = isWhisper ? _.filter(props.personalDetails, (details) => _.includes(whisperedTo, details.login)) : [];
+    const whisperedToAccountIDs = props.action.whisperedToAccountIDs || [];
+    const isWhisper = whisperedToAccountIDs.length > 0;
+    const isMultipleParticipant = whisperedToAccountIDs.length > 1;
+    const isWhisperOnlyVisibleByUser = isWhisper && ReportUtils.isCurrentUserTheOnlyParticipant(whisperedToAccountIDs);
+    const whisperedToPersonalDetails = isWhisper ? _.filter(props.personalDetailsList, (details) => _.includes(whisperedToAccountIDs, details.accountID)) : [];
     const displayNamesWithTooltips = isWhisper ? ReportUtils.getDisplayNamesWithTooltips(whisperedToPersonalDetails, isMultipleParticipant) : [];
```

Comment:
NAB - could we update the first param of this function (in the function definition, not here) to be something like `participantDetails` or something, instead of `participants` as it currently is? `participants` used to be an array of emails, and is very different from any kind of "personal details"

Reply:
Updated!